region/maridia/inner-green.json

```diff
@@ -144,6 +144,10 @@
                       "HiJump",
                       "canSpringBallJumpMidAir"
                     ]}
+                  ]},
+                  {"and": [
```

Comment:
Given the room layout, this climb seems to only get you right above the bomb blocks, but no higher. That's enough to reach the top ledge if you have Gravity, but not for suitless.

I think it needs to be added separately in both scenarios, as an alternative for either `canDestroyBombWalls` or `canPassBombPassages`.

Reply:
this needs a `canSwim` or `canSuitlessMaridia` anyway as you are going through a water room